dcap-ql/Cargo.toml

```diff
@@ -22,27 +22,37 @@ name = "live_quote"
 required-features = ["test-sgx-flc"]
 
 [features]
+default = ["bindings"]
+# If set, will provide bindings to libsgx_dcap_ql. Otherwise, you just get the
+# DCAP types.
+bindings = ["dcap-ql-sys", "sgxs-loaders", "libc"]
 # If set, will link to libsgx_dcap_ql. Otherwise, will use the dynamic loader
 # to access the library at runtime.
-link = ["dcap-ql-sys/link"]
+link = ["dcap-ql-sys/link", "bindings"]
 # Enable tests that can only be run on an SGX-enabled environment with FLC
-test-sgx-flc = []
+test-sgx-flc = ["bindings"]
+# Add the ability to verify quotes
+verify = ["mbedtls"]
 
 [dependencies]
 # Project dependencies
-"dcap-ql-sys" = { version = "0.2.0", path = "../dcap-ql-sys" }
-"sgxs-loaders" = { version = "0.2.0", path = "../sgxs-loaders" }
+"dcap-ql-sys" = { version = "0.2.0", path = "../dcap-ql-sys", optional = true }
+"sgxs-loaders" = { version = "0.2.0", path = "../sgxs-loaders", optional = true }
 "sgx-isa" = { version = "0.3.0", path = "../sgx-isa" }
 
 # External dependencies
 byteorder = "1.1.0" # Unlicense/MIT
 failure = "0.1.1"   # MIT/Apache-2.0
 lazy_static = "1"   # MIT/Apache-2.0
-libc = "0.2"        # MIT/Apache-2.0
+libc = { version = "0.2", optional = true }        # MIT/Apache-2.0
+"mbedtls" = { version = "0.5.1", default-features = false, optional = true }
+num = "0.2"
 num-derive = "0.2"  # MIT/Apache-2.0
 num-traits = "0.2"  # MIT/Apache-2.0
 serde = { version = "1.0.104", features = ["derive"], optional = true } # MIT/Apache-2.0
+yasna = { version = "0.3", features = ["num-bigint", "bit-vec"] }
```

Comment:
Num, yasna are added in this PR, presumably as part of the verification code? If so, they should be optional and be enabled by the verify feature.